eng/pipelines/docindex.yml

```diff
@@ -33,6 +33,8 @@ jobs:
       - template: /eng/pipelines/templates/steps/mvn-linux-repository-settings.yml
 
       - template: /eng/pipelines/templates/steps/install-rex-validation-tool.yml
+        parameters:
```

Comment:
Given you made this a global variable it isn't necessary to pass through to all the callers. Unless you think it needs to be overridden in some cases.

Reply:
I've updated to remove this to remove the parameter. I just wanted to wait for the pipelines I'd kicked off to finish their runs first.